mediapipe/graphs/instantmotiontracking/calculators/matrices_manager_calculator.cc

```diff
@@ -65,6 +66,8 @@ namespace {
 //  IMU_ROTATION - float[9] of row-major device rotation matrix [REQUIRED]
 //  USER_ROTATIONS - UserRotations with corresponding radians of rotation [REQUIRED]
 //  USER_SCALINGS - UserScalings with corresponding scale factor [REQUIRED]
+//  GIF_ASPECT_RATIO - Aspect ratio of GIF image used to dynamically scale GIF asset
+//  [REQUIRED]
```

Comment:
We should probably have this be an optional stream, and just default to 1.0f if it's not used.  But this is fine for now, since we're the only users.

Reply:
Agree that this should be an optional stream. Let's do it here.